### ISSUE TITLE
Top bar doesn't hide when disabling lock screen on laptop

### ISSUE THREAD

**Issue by notano** [mlutfy/hidetopbar#183]
After awaking from suspend or sleep, top bar revive and cover my window.

**Comment by tuxor1337** [comment]
Please give some more information about your setup:

* Are you using Wayland or X11?
* Which version of GNOME do you use?
* Can you reproduce this with all other GNOME extensions deactivated?
* What are your hide top bar preferences?

**Comment by notano** [comment]
Sorry for my lack of explanation.

I'm using default hide top bar settings on Ubuntu 19.04 on Wayland and can reproduce this with all my GNOME extensions deactivated. 

But I had same issue on all GNOME distribution I used(opensuse, arch, etc...). Couldn't you reproduce this disabling lock screen on laptop?

**Comment by tuxor1337** [comment]
No, I can't reproduce this in Fedora 30 on Wayland with GNOME 3.32.2. However, I still don't know your GNOME version and your preferences.

**Comment by notano** [comment]
Today I reinstalled Ubuntu and found out that hide top bar won't work with following extension.
https://github.com/lgpasquale/gnome-shell-extension-disable-screenshield/

Installing this extension and hide top bar and disabling Lock Screen on Suspend (Settings ➞ Privacy ➞ Screen Lock) may be able to reproduce this issue on your laptop.

I'm using GNOME 3.32.2. and default hide top bar preferences.

**Comment by notano** [comment]
I'm using GNOME 3.32.2. and default preferences.
I can reproduce this by disabling Lock Screen on Suspend(Settings ➞ Privacy ➞ Screen Lock) and closing my laptop lid.(suspend on terminal can't reproduce this... I'm sorry for confusing you.)
